templates/_linux.yaml

```diff
@@ -102,7 +102,7 @@ objects:
         - cloudInitNoCloud:
             userData: |-
               #cloud-config
-              password: {{ password }}
+              password: ${PASSWORD}
```

Comment:
As we discussed, please change this variable name to ${CLOUD_USER_PASSWORD}

Reply:
changed